qlf_fasm_db_builder.py

```diff
@@ -158,15 +186,35 @@ def main():
     assert xml_bitstream is not None and xml_bitstream.tag == "fabric_bitstream"
 
     # Parse the bitstream, group bits and features
-    grouped_bits = parse_fabric_bitstream(xml_bitstream)
+    grouped_bits, regions = parse_fabric_bitstream(xml_bitstream)
 
     # Count bits
+    print("Counting bits...")
```

Comment:
Might be good to start using the logger here as well

Reply:
Done.